lib/std/http/Server.zig

```diff
@@ -208,9 +208,9 @@ pub const Request = struct {
                     else => {},
                 }
 
-                var line_it = mem.splitSequence(u8, line, ": ");
+                var line_it = mem.splitScalar(u8, line, ':');
                 const header_name = line_it.next().?;
-                const header_value = line_it.rest();
+                const header_value = mem.trim(u8, line_it.rest(), " \t");
```

Comment:
where is the test coverage for this?

Reply:
I was trying to figure out where it fit best exactly, I decided to add another test block for checking server -> client headers.

Reply:
It's a pure function, so by far the best choice is to add a unit test that passes a string and then checks the resulting struct. Please don't waste the developer's time with an end-to-end test for this.